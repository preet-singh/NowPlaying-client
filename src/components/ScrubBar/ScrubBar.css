@media screen and (min-width: 320px){
    .scrub-bar {
    height: 25px;
    background-color: hsl(84, 60%, 45%);
  }
}

@media screen and (min-width: 768px){
  .scrub-bar {
    height: 50px;
    background-color: rgb(182, 152, 251);
    min-width: 300px;
    width: 100%;
    max-width: 650px;
    margin-top: 10px;
  }
}