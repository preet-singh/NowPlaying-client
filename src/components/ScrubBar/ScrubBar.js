import React from 'react';
import './ScrubBar.css'

function ScrubBar(props) {
  return (
    <div className='scrub-bar'>

    </div>

  )
}

export default ScrubBar;