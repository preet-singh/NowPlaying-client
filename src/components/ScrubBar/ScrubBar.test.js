import React from 'react';
import ScrubBar from './ScrubBar.js';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`ScrubBar component`, () => {

  it('renders the ScrubBar', () => {
    const wrapper = shallow(<ScrubBar />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

})