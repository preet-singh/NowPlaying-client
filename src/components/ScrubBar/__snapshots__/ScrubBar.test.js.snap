// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrubBar component renders the ScrubBar 1`] = `
<div
  className="scrub-bar"
/>
`;
